packages/cashier/src/components/cashier-locked/cashier-locked-states/empty-state-cashier-locked-ask-authenticate.tsx

```diff
@@ -0,0 +1,21 @@
+import { localize, Localize } from '@deriv/translations';
+import React from 'react';
+import EmptyState from '../../empty-state';
```

Comment:
@george-usynin-binary Thanks for the tips 🙇🏻 

Regarding the second one, We usually have `organizeImports` and `sortMembers` enabled in VSCode which sorts the import and the members automatically, Is there any reason we prefer to do this manually? 🤔 Should I disable them? 🤔

Reply:
@george-usynin-binary For now I've disabled it and made the changes based on the [guideline](https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md), But I think we should discuss it with the team and consider using it for better DX.
In my opinion, Developers should not care about simple things like formatting the code or sorting the import statements if it can be automated.

cc: @nijil-binary @Bahar-FS 🙇🏻 

```json
"[typescriptreact]": {
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.codeActionsOnSave": {
    "source.fixAll": true,
    "source.organizeImports": true,
    "source.sortMembers": true
  }
},
"[typescript]": {
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.codeActionsOnSave": {
    "source.fixAll": true,
    "source.organizeImports": true,
    "source.sortMembers": true
  }
},
```

Reply:
@farzin-fs sure we can discuss it.